RSpec/label_spec.rb

```diff
@@ -0,0 +1,33 @@
+# require_relative '../Classes/Book/label'
+# require_relative '../Classes/item'
+# require_relative '../Classes/Book/book'
+
+require_relative '../Classes/Book/label'
+require_relative '../Classes/item'
+require_relative '../Classes/Book/book'
+
+# F:\new move nov\microverse2022\module4\week5\Catalog-of-My-Things\Classes\Book
+
+# unit tests for label class
+describe Label do
+  let(:label) { Label.new('title', 'color') }
+  let(:item) { Item.new('2009-06-01') }
+  let(:book) { Book.new('publisher', 'good', '2009-06-01') }
+
+  context '# add_item method' do
```

Comment:
Error occures when testing the lable RSpec. please fix the issues. 
![image](https://user-images.githubusercontent.com/86018834/227204267-12967003-99a6-4f21-a25d-4d10472b797b.png)